### PR TITLE
Translate "Fetch a Hack Clubber" workshop to Spanish

### PR DESCRIPTION
**Pull request by davidballezaa**
- Added es-xl file with spanish translation
- Added `locales: 'es-xl'` line to README.md
- Fixed typos at README.md

### CODE REVIEW

File: workshops/fetch_a_hack_clubber/es-xl.md [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+
+Un gran dato conlleva una gran oportunidad. Hay mucho que podemos hacer con estos datos, ¡así que experimenta con ellos!
+
+Si no sabes qué hacer, aquí hay algunos ejemplos de siguientes pasos:
+
+* Usando el CSS personalizado de Scrapbook, ¡[esta versión del proyecto](https://meet-hackclub.vercel.app) hace cada perfil más personal!
+* ¿Quieres obtener un usuario específico? Sam Poder construyó una versión que te permite hacerlo si agregas un nombre de usuario a la URL, [prueba obteniendo a Sam aquí](https://meet-hackclub.vercel.app/sampoder). 
+* ¿Qué otros datos podemos conseguir? Podemos obtener una publicación del Scrapbook de alguien, ¡tal como [Sam hizo aquí](https://meet-hackclub.vercel.app/post-ver)!
+
+¿Estás haciendo algo cool? ¡Fascinate! ¡Compártelo en [#ship](https://hackclub.slack.com/archives/C0M8PUPU6/) en Slack y taggea a [@sampoder](https://hackclub.slack.com/archives/DT08DHJKF/)!
```
Comment (J-cordz) [inline workshops/fetch_a_hack_clubber/es-xl.md:250]: Hi @davidballezaa! I would suggest to translate "Awesome" as `Asombroso` , besides that everything looks great!

**Review by J-cordz — CHANGES_REQUESTED** [review]

File: workshops/fetch_a_hack_clubber/es-xl.md (hunk 1 above)
Comment (davidballezaa) [reply workshops/fetch_a_hack_clubber/es-xl.md]: Done, thank you!

Comment (J-cordz) [reply workshops/fetch_a_hack_clubber/es-xl.md]: Awesome! Thanks for the help!

**Review by sampoder — APPROVED** [review]
Thanks so much!

**Pull request merged**